frontend/appflowy_flutter/lib/plugins/ai_chat/chat_page.dart

```diff
@@ -273,10 +278,26 @@ class _ChatContentPage extends StatelessWidget {
           chatId: view.id,
           refSourceJsonString: refSourceJsonString,
           isLastMessage: isLastMessage,
-          onSelectedMetadata: (metadata) {
-            context
-                .read<ChatSidePanelBloc>()
-                .add(ChatSidePanelEvent.selectedMetadata(metadata));
+          onSelectedMetadata: (metadata) async {
+            if (isURL(metadata.name)) {
+              late Uri uri;
+
+              try {
+                uri = Uri.parse(metadata.name);
+                // `Uri` identifies `localhost` as a scheme
+                if (!uri.hasScheme || uri.scheme == 'localhost') {
+                  uri = Uri.parse("http://${metadata.name}");
+                  await InternetAddress.lookup(uri.host);
```

Comment:
If the address is not found, will this function throw an error?

By the way, I saw this openlink logic in other files. We may move it to an extension and utility file to share the same logic.

Reply:
> If the address is not found, will this function throw an error?

No, the exception will be caught and do nothing.

> By the way, I saw this open link logic in other files. We may move it to an extension and utility file to share the same logic.

This one is copied from the database version, but without searching google if it cannot be launched. I don't use `afLaunchUrl` doesn't work for all the cases I need it to. We can unify later if you wish.